tests/integration/actions/config/base.py

```diff
@@ -74,11 +77,21 @@ def test(self, request, tmux_session, step):
 
         if step.mask:
             # mask out some configuration that is subject to change each run
-            mask = "X" * 50
-            maskables = ["BECOME_PLUGIN_PATH", "CACHE_PLUGIN_CONNECTION", "COLLECTIONS_PATHS"]
-            for idx, line in enumerate(received_output):
-                if any(m in line for m in maskables):
-                    received_output[idx] = mask
+            maskables = [
+                "BECOME_PLUGIN_PATH",
+                "CACHE_PLUGIN_CONNECTION",
+                "COLLECTIONS_PATHS",
+                "DEFAULT_CALLBACK_PLUGIN_PATH",
+                "DEFAULT_LOCAL_TMP",
+            ]
+            # Determine if a menu if showing
+            match = re.search("CURRENT VALUE", received_output[0])
+            if match:
+                mask_start, mask_end = match.span()
+                mask = (mask_end - mask_start) * "X"
```

Comment:
Isn't this just
```suggestion
                mask = len("CURRENT VALUE") * "X"
```
?

Reply:
(which then could be moved outside the if-block?)

Reply:
I prefer using the results of the span in this case rather than repeating the string..............

Reply:
(probably b/c it's a reusable pattern in case the search was done against a regex)

Reply:
I think I'd opt to not use regex here at all, it's quite a hammer, just to search for a simple string and roughly an order of magnitude slower:

```

In [8]: %timeit "CURRENT VALUE" in "FOO     BAR      CURRENT VALUE     BAZ"
38.6 ns ± 0.169 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)

In [9]: %timeit re.search("CURRENT VALUE", "FOO     BAR      CURRENT VALUE     BAZ")
360 ns ± 0.847 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)
```

Reply:
Ah, nevermind, you use `mask_start` below.

Reply:
Since it was a test, I wasn't all too concerned about the re, we do need to pay attention to those in the user facing code....  But now that you mention it, I think str.find would work.  

let's see how it looks

Reply:
Yeah, I was just going to come back and suggest that.

```
In [10]: %timeit "FOO     BAR      CURRENT VALUE     BAZ".find("CURRENT VALUE")
87.2 ns ± 0.126 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)
```

Reply:
And I see @webknjaz beat me to the suggestion anyway :) I'll show myself out.

Reply:
I've added an implementation suggestion not using `re` below: https://github.com/ansible/ansible-navigator/pull/871/files#r799593279

Reply:
thanks for the help here @relrod and @webknjaz